### PR TITLE
fix: replace `Duration` with `u64` in schema

### PR DESCRIPTION
**Pull request by mversic**
## Description

* replace `Duration` with `u64` in `schema.json`
* enable registering time trigger in genesis

### Linked issue

Closes #{issue_number} 

### Checklist

- [ ] I've read `CONTRIBUTING.md`
- [ ] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

File: core/src/smartcontracts/isi/triggers/mod.rs [hunk 1]
```diff
@@ -65,20 +67,21 @@ pub mod isi {
...
+                            // Gensis block
                             None => {
-                                return Err(Error::InvalidParameter(
-                                    InvalidParameterError::TimeTriggerInThePast,
-                                ));
+                                if schedule.start_ms < genesis_creation_time_ms {
+                                    return Err(Error::InvalidParameter(
+                                        InvalidParameterError::TimeTriggerInThePast,
+                                    ));
+                                }
```
Comment (mversic) [inline core/src/smartcontracts/isi/triggers/mod.rs:76]: now it's allowed to register time trigger in genesis

**Comment by Erigara** [comment]
What is your motivation for removing `Duration` from schema?

**Comment by mversic** [comment]
> What is your motivation for removing `Duration` from schema?

because it was a pain to use and incorrect. In schema we had an entry:
```json
"Duration": {                                                          
  "Tuple": [                                                           
    "u64",                                                             
    "u32"                                                              
  ]                                                                    
}
```

but in `genesis.json` it was:
```json

"filter": {
  "Time": {
    "Schedule": {
      "start": {
        "secs": 1790774026,
        "nanos": 0
      },
      "period": {
        "secs": 1,
        "nanos": 0
      }
    }
  }
}
```

we can consider restoring it but it has to be done properly

**Review by Erigara — DISMISSED** [review]

**Review by nxsaken — APPROVED** [review]

File: core/src/smartcontracts/isi/triggers/mod.rs [hunk 2]
```diff
@@ -65,20 +67,21 @@ pub mod isi {
                 TriggeringEventFilterBox::Time(time_filter) => {
                     if let ExecutionTime::Schedule(schedule) = time_filter.0 {
                         match last_block_estimation {
-                            // We're in genesis
+                            // Gensis block
```
Comment (dima74) [inline core/src/smartcontracts/isi/triggers/mod.rs:70]:
```suggestion
                            // Genesis block
```

**Review by dima74 — APPROVED** [review]

**Review by dima74 — APPROVED** [review]

**Review by nxsaken — APPROVED** [review]

**Pull request merged**